proto/lighthouse-result.proto

```diff
@@ -144,6 +144,21 @@ message LighthouseResult {
 
     // How Lighthouse was run, e.g. from the Chrome extension or from the npm module
     string channel = 4;
+
+    // The possible form factors an audit can be run in
+    enum DeviceScreenEmulationMethod {
```

Comment:
Is this really necessary in proto?  LR will always use one option?  Similar to throttling settings is always simulated?  We elide much of config for this reason iirc.

Reply:
nah. we can leave out of the proto for now. i forgot that we only did a subset.